modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/mds/channel/TriggerEvent.java

```diff
@@ -35,7 +35,9 @@
     private Channel channel;
 
     @Field
+    @JsonIgnore
```

Comment:
Why do we need JsonIgnore here? It seems that Tasks will stop displaying the available parameters for the chosen trigger if we do that.